js_modules/dagster-ui/packages/ui-core/src/asset-graph/useAssetGraphData.tsx

```diff
@@ -249,12 +249,6 @@ export function useAssetGraphData(opsQuery: string, options: AssetGraphFetchScop
   };
 }
 
-const computeGraphData = indexedDBAsyncMemoize<GraphDataState, typeof computeGraphDataWrapper>(
-  computeGraphDataWrapper,
-  'computeGraphData',
-  (props) => hashObject({props, version: 3}),
-);
-
```

Comment:
the indexeddb caching here was adding extra overhead that was greater than the work it was saving us from...